src/js/player.js

```diff
@@ -1979,7 +1979,7 @@ class Player extends Component {
   documentFullscreenChange_(e) {
     const fsApi = FullscreenApi;
 
-    this.isFullscreen(document[fsApi.fullscreenElement] === this.el());
+    this.isFullscreen(document[fsApi.fullscreenElement] === this.el() || this.el().matches(':' + fsApi.fullscreen));
```

Comment:
I was going to comment about how on IE11 `matches` is `msMatchesSelector` but I think we'll never actually get here on IE11

Reply:
Looks like I was wrong :(